src/FSharpLint.Core/Rules/Formatting/TupleFormatting/TupleCommaSpacing.fs

```diff
@@ -11,31 +11,46 @@ open FSharpLint.Rules.Helper
 
 // Check for single space after commas in tuple.
 let checkTupleCommaSpacing (args:AstNodeRuleParams) (tupleExprs:SynExpr list) tupleRange _ =
+    let choose (expr: SynExpr) (nextExpr: SynExpr) = 
+      if expr.Range.EndLine = nextExpr.Range.StartLine && expr.Range.EndColumn + 2 <> nextExpr.Range.StartColumn then
+        let commaRange = Range.mkRange String.Empty expr.Range.End nextExpr.Range.Start
+        let map commaText =
+            lazy
+                (Some
+                    {
+                        FromRange = commaRange
+                        FromText = commaText
+                        ToText = ", "
+                    })
```

Comment:
@webwarrior-ws this would be more readable this way:

```
{
    FromRange = commaRange
    FromText = commaText
    ToText = ", "
}
|> Some
|> lazy
```

Reply:
then how about:

```
lazy <|
    Some <|
        {
            FromRange = commaRange
            FromText = commaText
            ToText = ", "
        }
```

The key is to avoid the parens